genai-perf/genai_perf/wrapper.py

```diff
@@ -129,11 +131,9 @@ def build_cmd(args: Namespace, extra_args: Optional[List[str]] = None) -> List[s
 
         utils.remove_file(args.profile_export_file)
 
+        # TODO: Temp, remove below change
```

Comment:
placeholder comment (to not forget to remove this)

Reply:
Good catch, removed.